industrial_ci/src/docker.sh

```diff
@@ -111,7 +111,10 @@ function ici_run_cmd_in_docker() {
   wait %% || ret=$?
   trap - INT
   if [ -n "$commit_image" ]; then
-    docker commit -m "$_COMMIT_IMAGE_MSG" "$cid" $commit_image > /dev/null
+    if [ "use_containerid" != "$commit_image" ]; then
```

Comment:
The effect could be achieved by passing an empty string as `DOCKER_COMMIT`.
(but the length check in line 113 needs to be updated)
In addtion, `use_containerid`  is really misleading, because the container is not used, but the image will be untagged.

IMHO it is a bad idea to commit without a tag, it just adds lot anonymous images.

Reply:
Talked to @AustinDeric offline. He thinks we can remove this logic. I'll add the change in  https://github.com/AustinDeric/industrial_ci/pull/3